src/renderer/dx/DxRenderer.cpp

```diff
@@ -681,6 +677,18 @@ Microsoft::WRL::ComPtr<IDXGISwapChain1> DxEngine::GetSwapChain()
     return _dxgiSwapChain;
 }
 
+til::rectangle DxEngine::_InvalidToFullRow(const til::rectangle& rc) const
+{
+    if (_invalidateFullRows)
+    {
+        return til::rectangle{ til::point{static_cast<ptrdiff_t>(0), rc.top()}, til::size{_invalidMap.size().width(), rc.height()} };

```

Comment:
bah is that cast really required

Reply:
They have to be the same type as I don't have a constructor that mixes them... and it's not autopromoting...